go.mod

```diff
@@ -23,7 +23,7 @@ require (
 	github.com/spf13/cobra v0.0.5
 	github.com/spf13/pflag v1.0.3
 	github.com/stretchr/testify v1.8.2
-	github.com/uselagoon/machinery v0.0.14
+	github.com/uselagoon/machinery v0.0.15-0.20231214035056-b3e9d4c4b4dd
```

Comment:
Now that `0.0.15` is released, this will need to be updated to use it, then we can merge this